test/rss-aws-developer-be.test.ts

```diff
@@ -0,0 +1,22 @@
+import { handler as getProductsList } from '../lambda/getProductsList';
+import { handler as getProductsById } from '../lambda/getProductsById';
+
+describe('Products service', () => {
+  it('should return 200 statusCode for getProductsList', async () => {
+    const response = await getProductsList({});
+
+    expect(response.statusCode).toEqual(200);
+  });
+
+  it('should return 200 statusCode for getProductsById', async () => {
+    const response = await getProductsById({ pathParameters: { productId: '3' } });
+
+    expect(response.statusCode).toEqual(200);
+  });
+
+  it('should return 404 statusCode for getProductsById', async () => {
+    const response = await getProductsById({ pathParameters: { productId: '4' } });
```

Comment:
this is a fragile test. when you add a product with id '4', this test will fail.

slightly better:
```suggestion
    const response = await getProductsById({ pathParameters: { productId: '-404' } });
```